cmd/krew/cmd/index.go

```diff
@@ -62,11 +67,65 @@ var indexAddCmd = &cobra.Command{
 	Example: "kubectl krew index add default " + constants.IndexURI,
 	Args:    cobra.ExactArgs(2),
 	RunE: func(_ *cobra.Command, args []string) error {
-		return indexoperations.AddIndex(paths.IndexBase(), args[0], args[1])
+		return indexoperations.AddIndex(paths, args[0], args[1])
 	},
 }
 
+var indexDeleteCmd = &cobra.Command{
+	Use:   "remove",
+	Short: "Remove a configured index",
+	Long: `Remove a configured index repository which is used to discover and
+install plugins from.
+
+If there are plugins installed from the specified index, the index cannot be
+removed (unless --force) is used. Removing indexes while there are plugins
+installed from them is not supported behavior.`,
```

Comment:
👍🏼

Reply:
Done.

Reply:
I meant more about the "unsupported behavior" part. I feel like since there's a `--force` option it is supported, but it might result in unsupported behavior.

Reply:
Please give exact copy and I'll incorporate.

Reply:
My vote is for:
```
Remove a configured index repository which is used to discover and
install plugins from.

If there are plugins installed from the specified index, the index cannot be
removed (unless --force is used). Removing indexes while there are plugins
installed from them may result in unsupported behavior.
```

I like mentioning "unsupported behavior" because it conveys the not recommended part while alluding to potential consequences of performing this action.

Reply:
I suggest the following:

> Removes a plugin index.
>
> It is only safe to remove indexes without installed plugins. Removing an index while there are plugins installed will result in an error unless the `--force` option is used. This is not recommended.

Reply:
Done.